src/content/docs/infrastructure/prometheus-integrations/integrations-list/calico-integration.mdx

```diff
@@ -6,3 +6,95 @@ tags:
   - Prometheus
   - Kubernetes
 ---
+
+import kubernetesCalicoDashboard from 'images/kubernetes_screenshot-crop_calico-dashboard.png'
+
+[Calico CNI](https://github.com/projectcalico/calico) is a widely adopted, battle-tested open source networking, and network security solution for Kubernetes, virtual machines, and bare-metal workloads. Calico provides two major services for Cloud Native applications:
+
+* Network connectivity between workloads.
+* Network security policy enforcement between workloads.
+
+Use New Relic to view a dashboard based on Prometheus metrics which helps you understand the Calico CNI of your k8s cluster. With New Relic you can monitor:
+
+* Active network policies by instances
+* Active IP tables rules by instances and chain type (filter, mangle, nat, and raw)
+* IP tables save and restore errors
+* BPF specific metrics if BPF is used as a dataplane for Calico
+
+<img
+  title="Calico Dashboard"
+  alt="Calico Dashboard"
+  src={kubernetesCalicoDashboard}
+/>
+
+## Enable the integration
+
+Follow these steps to enable the integration.
+
+1. Follow the [Calico documentation](https://projectcalico.docs.tigera.io/maintenance/monitor/monitor-component-metrics) for Prometheus to discover the Calico metrics endpoints.
+
+2. Set up Prometheus monitoring. Prometheus metrics needs to be integrated with New Relic, you can use the Prometheus Agent or the Remote Write integration, see [how to send Prometheus metrics](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/get-started/send-prometheus-metric-data-new-relic/).
+
+<Callout variant="Prometheus Agent ONLY!">
+Prometheus Agent [by default](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent#scrape-metrics-only-from-prometheus-integrations-scrape-prometheus-integrations) only scrapes metrics from a [set of integrations](/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro).
+
+If this is the case make sure your pod or endpoint can be identifed with one of the following labels "app.kubernetes.io/name", "app.newrelic.io/name", "k8s-app" containing the string "calico"
+</Callout>
+
+3. Use the following query to confirm metrics are being ingested as expected:
+
+    ```sql
+    FROM Metric SELECT count(*) WHERE metricName LIKE 'felix_%' FACET metricName LIMIT MAX
+    ```
+
+4. Install the [Calico quickstart](https://newrelic.com/instant-observability/calico-newrelic) to access built-in dashboards and and [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts/).
+
+  Once you imported, you can edit or clone the assets to adapt them to your specific requirements.
+
+<Callout title="Important!">
+
+  Some charts of the dashboard include queries with conditions that requires yourpod or endpoint to be identifed with one of the following labels "app.kubernetes.io/name", "app.newrelic.io/name", "k8s-app" containing the string "calico"
```

Comment:
- `yourpod` missing space
 - with conditions that require`s` (I think we should drop the s)

Reply:
Fixed